controllers/provider.go

```diff
@@ -103,8 +103,8 @@ func (c *ApiController) UpdateProvider() {
 		return
 	}
 
-	c.Data["json"] = wrapActionResponse(object.UpdateProvider(id, &provider))
-	c.ServeJSON()
+	response := wrapActionResponse(object.UpdateProvider(id, &provider))
```

Comment:
Do you test the frontend can handle the response successfully? The wrapActionResponse has wraped the data. If ResponseOk rewrap it, frontend may can not handle the response.

Reply:
Now I have found this problem and I am working on it. Thank you！

Reply:
> Now I have found this problem and I am working on it. Thank you！

Are you done with this issue？How is it going now